src/main/kotlin/com/github/avrokotlin/avro4k/schema/ClassSchemaFor.kt

```diff
@@ -118,6 +119,7 @@ class ClassSchemaFor(
       val default: Any? = annos.default()?.let {
          when {
              it == Avro.NULL -> Schema.Field.NULL_DEFAULT_VALUE
+             schemaWithResolvedNamespace.extractNonNull().type == Schema.Type.ENUM -> annos.default()
```

Comment:
You can just add it to the list in the following line as "it" is annos.default()

Reply:
Moved this check to use same list check